WalletWasabi.Fluent/Styles/ScrollViewer.axaml

```diff
@@ -0,0 +1,113 @@
+							   Grid.Row="1"
+							   Focusable="False" />
+					<ScrollBar Name="PART_VerticalScrollBar"
+							   AllowAutoHide="{TemplateBinding AllowAutoHide}"
+							   Orientation="Vertical"
+							   LargeChange="{Binding LargeChange.Height, RelativeSource={RelativeSource TemplatedParent}}"
+							   SmallChange="{Binding SmallChange.Height, RelativeSource={RelativeSource TemplatedParent}}"
+							   Maximum="{TemplateBinding VerticalScrollBarMaximum}"
+							   Value="{TemplateBinding VerticalScrollBarValue, Mode=TwoWay}"
+							   ViewportSize="{TemplateBinding VerticalScrollBarViewportSize}"
+							   Visibility="{TemplateBinding VerticalScrollBarVisibility}"
+							   Grid.Column="1"
+							   Focusable="False" />
+					<Panel x:Name="PART_ScrollBarsSeparator" Grid.Row="1" Grid.Column="1" Background="{DynamicResource ScrollViewerScrollBarsSeparatorBackground}" />
+
+					<!-- Scroll hints -->
+					<Viewbox Classes="ScrollViewerCuePath" VerticalAlignment="Top">
+						<Viewbox.IsVisible>
+							<MultiBinding Converter="{x:Static converters:MenuScrollingVisibilityConverter.Instance}"
+										  ConverterParameter="0">
+								<Binding RelativeSource="{RelativeSource TemplatedParent}" Path="VerticalScrollBarVisibility"/>
+								<Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Offset.Y"/>
+								<Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Extent.Height"/>
+								<Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Viewport.Height"/>
+							</MultiBinding>
+						</Viewbox.IsVisible>
+						<Path Data="M 0 4 L 8 4 L 4 0 Z" Classes="ScrollViewerCuePath"/>
+					</Viewbox>
+
+					<Viewbox Classes="ScrollViewerCuePath" Stretch="Fill" Grid.ColumnSpan="2" Grid.Row="0" Grid.Column="0" VerticalAlignment="Bottom" >
+						<Viewbox.IsVisible>
+							<MultiBinding Converter="{x:Static converters:MenuScrollingVisibilityConverter.Instance}"
+										  ConverterParameter="100">
+								<Binding RelativeSource="{RelativeSource TemplatedParent}" Path="VerticalScrollBarVisibility"/>
+								<Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Offset.Y"/>
+								<Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Extent.Height"/>
+								<Binding RelativeSource="{RelativeSource TemplatedParent}" Path="Viewport.Height"/>
+							</MultiBinding>
+						</Viewbox.IsVisible>
+						<Path Data="M 0 0 L 4 4 L 8 0 Z" Classes="ScrollViewerCuePath" />
+					</Viewbox>
+				</Grid>
+
+			</ControlTemplate>
+		</Setter>
+	</Style>
+
+	<Style Selector="Path.ScrollViewerCuePath">
+		<Setter Property="Stroke" Value="{StaticResource SystemBaseMediumLowColor}" />
+		<Setter Property="Fill" Value="{StaticResource SystemBaseMediumHighColor}"></Setter>
+		<Setter Property="StrokeThickness" Value="1" />
+	</Style>
+
+	<Style Selector="Viewbox.ScrollViewerCuePath">
+		<Setter Property="Width" Value="25" />
+		<Setter Property="Height" Value="10" />
```

Comment:
These are the only differences I could spot with #7326. Why was it reverted?

Reply:
> These are the only differences I could spot with #7326. 

Yes, arrows are slightly bigger now!

> Why was it reverted?

I merged it to master accidentally 🤦‍♂️